resources/oracle-to-psql-converters/data-files/organisations-data.txt

```diff
@@ -0,0 +1,8 @@
+REM INSERTING into EXPORT_TABLE
+SET DEFINE OFF;
+Insert into EXPORT_TABLE (SBI,ADDRESSLINE1,ADDRESSLINE2,ADDRESSLINE3,CITY,COUNTY,POSTCODE,EMAILADDRESS,FRN,NAME,UPDATED) values (106613073,'COOGIE FARM','CHURCH END','CAWOOD','SELBY',null,'Z03 3PN','email1@yahoo.co.uk',1102358109,'FARMNAME1',to_date('14-FEB-24 01:34:17','DD-MON-YY HH:MI:SS'));
+Insert into EXPORT_TABLE (SBI,ADDRESSLINE1,ADDRESSLINE2,ADDRESSLINE3,CITY,COUNTY,POSTCODE,EMAILADDRESS,FRN,NAME,UPDATED) values (106268747,'OETER COTTAGE',null,'HAMSTERLEY','BISHOP AUCKLAND',null,'CC13 3KA','email12@graysllp.co.uk',1101007842,'FARMNAME12',to_date('14-FEB-24 01:34:17','DD-MON-YY HH:MI:SS'));

```

Comment:
I still have concerns that the email addresses and addresses being obscured still have data we don't want to show